test/fetch/request.js

```diff
@@ -13,7 +13,9 @@ const {
   Blob: ThirdPartyBlob,
   FormData: ThirdPartyFormData
 } = require('formdata-node')
-const hasSignalReason = !!~process.version.localeCompare('v16.14.0', undefined, { numeric: true })
+const { nodeMajor } = require('../../lib/core/util')
+
+const hasSignalReason = (nodeMajor > 16 || (nodeMajor === 16 && nodeMajor > 14))
```

Comment:
This might be related to the v16.8 CI failure,
```
     # Subtest: pre aborted signal
        ok 1 - should be equal
        not ok 2 - should be equal
          ---
          found: null
          wanted: gwak
          compare: ===
          at:
            line: 285
            column: 7
            file: test/fetch/request.js
            type: Test
          stack: |
            Test.<anonymous> (test/fetch/request.js:285:7)
          source: |2
              if (hasSignalReason) {
                t.equal(req.signal.reason, 'gwak')
            ------^
              }
              t.end()
          ...
        
        1..2
        # failed 1 of 2 tests
    not ok 16 - pre aborted signal # time=14ms
```

Reply:
That made the trick, thanks!